MessageProvider.hpp

```diff
@@ -0,0 +1,39 @@
+#ifndef MESSAGE_PROVIDER_HPP_
```

Comment:
Good approach to maintain separate file for language. this allows easy extension and also the provision to select language dynamically!!